CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 3.3.1)
+
+project(pms CXX)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Add `set(CMAKE_CXX_EXTENSIONS OFF)` after this line (don't want gnutensions, less portable)